go/tasks/plugins/hive/execution_state.go

```diff
@@ -344,11 +345,13 @@ func KickOffQuery(ctx context.Context, tCtx core.TaskExecutionContext, currentSt
 
 	taskExecutionIdentifier := tCtx.TaskExecutionMetadata().GetTaskExecutionID().GetID()
 	commandMetadata := client.CommandMetadata{TaskName: taskName,
-		Domain:        taskExecutionIdentifier.GetTaskId().GetDomain(),
-		Project:       taskExecutionIdentifier.GetNodeExecutionId().GetExecutionId().GetProject(),
-		Labels:        tCtx.TaskExecutionMetadata().GetLabels(),
-		AttemptNumber: taskExecutionIdentifier.GetRetryAttempt(),
-		MaxAttempts:   tCtx.TaskExecutionMetadata().GetMaxAttempts(),
+		Domain:              taskExecutionIdentifier.GetTaskId().GetDomain(),
```

Comment:
I know this existed before, but it would be great if you can fix it as well to:
```suggestion
		Domain:              taskExecutionIdentifier.GetNodeExecutionId().GetExecutionId().GetDomain(),
```

Reply:
addressed